packages/binding-opcua/src/opcua-client.ts

```diff
@@ -163,7 +163,14 @@ export default class OpcuaClient implements ProtocolClient {
         if (result.statusCode === StatusCodes.BadNodeIdUnknown) {
             throw new Error("Invalid nodeId");
         }
-        const body = JSON.stringify(result.toJSON());
+        const body = JSON.stringify(result, (k, v) => {
```

Comment:
In fact, We should use const body = JSON.stringify(result.toJSON(), (k,v)=> ArrayBuffer.isView(v as ArrayLike<unknown>)? Array.from(v) : v);